bin/curriculum/import_unit_details.rb

```diff
@@ -127,7 +127,7 @@ def main(options)
       lesson_group.update_from_curriculum_builder(cb_chapter) if options.models.include?('LessonGroup')
     end
 
-    script.fix_script_level_positions
+    script.fix_script_level_positions(true)
     script.update!(show_calendar: !!cb_unit['show_calendar'], is_migrated: true)
```

Comment:
Could you just flip these two lines instead of adding a parameter to `fix_script_level_positions`?

Reply:
+1

Reply:
Done